python_envs/install_python

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+set -vx
+prefix=${prefix:-$HOME/python_envs}
+name=${name:-easy}
+yaml_file=${yaml_file:-environment.yaml}
+
+
+conda deactivate
+mamba deactivate
+deactivate
+
+set -e
+
+module purge
+module load git
+
+# nvidia-smi || ( >&2 echo  "need a gpu node to get the correct pytorch version" ; exit 1 )
+
+git_version=$(git rev-parse --verify HEAD)
+logname="hackathon_env_${git_version}_$(date +%F_%T)_install.log"
+
+if [ ! -d  ${prefix}/miniconda3 ] ; then
+mkdir -p ${prefix}/miniconda3
+wget https://github.com/conda-forge/miniforge/releases/latest/download/Miniforge3-Linux-x86_64.sh -O ${prefix}/miniconda3/miniconda.sh
+bash ${prefix}/miniconda3/miniconda.sh -b -u -p ${prefix}/miniconda3
+rm -f ${prefix}/miniconda3/miniconda.sh
+fi
+
+${prefix}/miniconda3/bin/conda update -n base -c conda-forge conda mamba
```

Comment:
So we also install a new miniconda if, e.g. micromamba, is already installed?

Reply:
yes, no trust in things that are on our system. ;)

Reply:
I second that! 😉 

Maybe we could even go the full way and have a script that sets up the environment on an empty VM. And other centres could base their installation on that script? This way we could also test the installation in Github CI/CD